### PR TITLE
Ingest intl csvs

### PR DESCRIPTION
**Pull request by lgarvey**

### CODE REVIEW

File: dnb_worldbase/ingest.py [hunk 1]
```diff
@@ -63,34 +69,110 @@ def update_company(wb_data):
...
         wb_data = dict(zip(WB_HEADER_FIELDS, row_data))
 
         try:
-            created = update_company(wb_data)
+            created = update_company(wb_data, _creation_date)
+
+        except(KeyboardInterrupt, SystemExit):
+            raise
+
         except BaseException as ex:
```
Comment (oisinmulvihill) [inline dnb_worldbase/ingest.py:98]: Is this no just Exception as ex instead of BaseException as ex to catch all?

File: dnb_worldbase/ingest.py [hunk 2]
```diff
@@ -63,34 +69,110 @@ def update_company(wb_data):
...
+        csv_class = WBIntlCsvProcessor
+    elif base_name.startswith('UK'):
+        csv_class = WBUKCsvProcessor
+    else:
+        raise ValueError(f'File name does not match expected format: {file_path}')
+
+    with open(file_path, 'rt', encoding='iso-8859-1') as wb_file:
+        stats = process_csv_data(csv_class(wb_file), creation_date)
+
+    return stats
```
Comment (oisinmulvihill) [inline dnb_worldbase/ingest.py:178]: Could you add test(s) to cover more of the ingest.py?

**Review by oisinmulvihill — COMMENTED** [review]
generally looks ok. I have a two questions below.

File: dnb_worldbase/ingest.py (hunk 1 above)
Comment (lgarvey) [reply dnb_worldbase/ingest.py:98]: Yep, baseexcept is superfluous here.  Will change it.

File: dnb_worldbase/ingest.py (hunk 2 above)
Comment (lgarvey) [reply dnb_worldbase/ingest.py:178]: OK, will do this.

**Comment by oliverjwroberts** [comment]
Confirmed with @lgarvey this can be closed out.

**Pull request closed without merging**